Scripts/Python/xOpeningSequence.py

```diff
@@ -399,6 +431,8 @@ def IStartGame(self):
         if entry is not None:
             entry.chronicleSetValue("yes")
             entry.save()
+        elif IsTutorialPath():
+            vault.addChronicleEntry(kIntroPlayedChronicle, 2, "no")
```

Comment:
I'm assuming this makes the Yeesha intro play once the player reaches Relto?

Reply:
> I'm assuming this makes the Yeesha intro play once the player reaches Relto?

Yup. On Cleft playthrough, the original Atrus intro plays to start, then, after getting Relto, the Yeesha intro plays (although there's been discussion on replacing the latter video with something newer).